{"uuid": "4b0cf1de-c03c-4e48-b88c-d766e5a57f86", "children": ["e3d187b1-ee19-4fbe-b259-d82bceade051"], "befores": [{"name": "with_open_lottery_page", "status": "passed", "start": 1568464804051, "stop": 1568464806775}], "afters": [{"name": "with_open_lottery_page::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\n\n", "trace": "  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 786, in _teardown_yield_fixture\n    next(it)\n  File \"d:\\coding\\case\\caipiao-client\\conftest.py\", line 14, in with_open_lottery_page\n    cpclotteryhall.close()\n  File \"d:\\coding\\base\\box.py\", line 922, in close\n    self.base_driver.close_browser()\n  File \"d:\\coding\\base\\box.py\", line 276, in close_browser\n    self._base_driver.close()\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 688, in close\n    self.execute(Command.CLOSE)\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\dajun\\documents\\caipiao\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1568464816152, "stop": 1568464816202}], "start": 1568464804051, "stop": 1568464816202}